src/lsp/cobol_ptree/statements_visitor.ml

```diff
@@ -829,8 +829,27 @@ and fold_accept_clause' (v: _ #folder) : accept_clause with_loc -> 'a -> 'a =
 and fold_accept_with_clause' (v: _ #folder) : accept_with_clause with_loc -> 'a -> 'a =
   handle' v#fold_accept_with_clause' v
     ~fold:begin fun _v clause x -> match clause with
-      | AcceptUpdate -> x
       | AcceptAttribute _ -> x
+      | AcceptAuto -> x
+      | AcceptFull -> x
+      | AcceptRequired -> x
+      | AcceptSecure -> x
+      | AcceptGrid -> x
+      | AcceptLeftLine -> x
+      | AcceptOverLine -> x
+      | AcceptSize ii -> fold_ident_or_intlit v ii x
+      | AcceptControl i -> fold_ident v i x
+      | AcceptPromptCharacter (inno) ->
```

Comment:
No need for the parentheses around `inno`

Reply:
I know, but I like them :p

Reply:
Actually I think they have a different semantics depending on the constructor: `type t = A of (int * string)` is not equivalent to `type t' = A of int * string`, and I see the additional pair of parentheses as similarly misleading.

Reply:
It's true when defining the constructor, but when using a constructor, it does not matter - hopefully the type helps disambiguate (otherwise, how could you tell the type by solely looking at `A (42, "toto")` ?)

Reply:
Sure the meaning is the same when matching, the difference is in type definitions.
In any case Boris and I voted the same, for not using unneeded parentheses whenever possible in such patterns.

Reply:
That has been changed already. I wonder why this is not showing as outdated.